pkg/kubelet/cm/node_container_manager_test.go

```diff
@@ -111,24 +111,64 @@ func TestNodeAllocatableReservationForScheduling(t *testing.T) {
 			assert.Equal(t, expected.MilliValue(), v.MilliValue(), "test case %d failed for resource %q", idx+1, k)
 		}
 	}
-}
 
-func TestNodeAllocatableWithNilHardThreshold(t *testing.T) {
```

Comment:
Remove this function because it is one of test cases in function `TestNodeAllocatableReservationForScheduling` above